src/app/src/sensorUtils.js

```diff
@@ -17,46 +29,87 @@ export function makeRiverGaugeRequest(id, isApiRequest) {
     return axios.get(url);
 }
 
-function parseCsvString(csvString) {
-    const data = Papa.parse(csvString, { header: true, comments: "#", dynamicTyping: true, skipEmptyLines: true });
-    return data;
+export function parseCsvString(csvString) {
+    const data = Papa.parse(csvString, {
+        header: true,
+        comments: '#',
+        dynamicTyping: true,
+        skipEmptyLines: true,
+    });
+
+    return data.data;
 }
 
-export function parseRiverGaugeApiData(id, apiData) {
-    const {
-        data: {
-            value: {
-                timeSeries: data,
-            },
+export function parseRiverGaugeApiData(id, data) {
+    const extractedVariableData = VARIABLES.reduce(
+        (acc, variable, idx) => {
+            const apiVariableData = data[idx];
+            if (apiVariableData) {
+                const sensorValue = Number(
+                    apiVariableData.values[0].value[0].value
```

Comment:
How sure are we that this chain of property lookups will never fail? We could consider using lodash get -- https://lodash.com/docs/4.17.11#get -- here to fallback to a default value if something breaks.

Reply:
It also might make sense to wrap the Number call in something like https://lodash.com/docs/4.17.11#defaultTo for cases when

- the property lookup chain succeeds
- the `Number(whatever)` call results in `NaN` for whatever reason

Reply:
Yeah, good call. It looks like the API response structures are pretty static, but this does look like trouble. I'll revise.

Reply:
On second thought I didn't modify the guts of this, it got changed by Prettier. Going to open an issue to make it a little more robust in case the API response changes.

Reply:
https://github.com/azavea/ism-watershed-wellness-snapshot/issues/65